CHANGELOG.md

```diff
@@ -28,7 +28,8 @@ Here is a template for new release sections
 ### Removed
 -
 ### Fixed
--
+- Skip `test_benchmark_KPI` as it was seen to be consumming the whole test time leading to timeout on github action (#826)
+- Reduce `simulation_settings.evaluated_period` to one day for the tests where simulation results are not important (for E0 and D2 test modules setup) (#826)
```

Comment:
You only hard-coded this, but did not change the input files to decrease the test data volume. Intentional?

Reply:
I did not know if the longer input files are needed for other purposes, this is why I only changed the evaluated period

Reply:
Okay, good to know!